### PR TITLE
Bug 4864 Missing Brokerage Live Settings

### LINKED ISSUE #4864: Missing Various Brokerages Live Algorithm Settings Classes
#### Expected Behavior

All brokerages have a LiveAlgorithmSetting subclass for their respective required settings.

#### Actual Behavior

Have a LiveAlgorithmSettings Subclass:
 - Default (QuantConnect)
 - FXCM
 - Interactive Brokers
 - Oanda
 - Tradier

Don't have a LiveAlgorithmSettings Subclass:
 - Bitfinex
 - Binance
 - GDAX
 - Alpaca
 - AlphaStreams

Without these there is no standardized approach to create live algorithm settings to launch a live algorithm through the API. To see what the implemented ones look like, go to 

Common/Api/LiveAlgorithmSettings.cs 

Reference `LiveAlgorithmSettings_CanBeCreated_Successfully` test for usage of these classes, it fails without these implemented.

#### Potential Solution

Create the required classes with their respective needs for api calls.

#### Reproducing the Problem

Run `LiveAlgorithmSettings_CanBeCreated_Successfully` from the api tests.

#### Checklist

- [x] I have completely filled out this template
- [x] I have confirmed that this issue exists on the current `master` branch
- [x] I have confirmed that this is not a duplicate issue by searching [issues](https://github.com/QuantConnect/Lean/issues)

- [x] I have provided detailed steps to reproduce the issue

### PR DESCRIPTION
**Pull request by C-SELLERS**
#### Description

Addition of the missing brokerage live settings to work with the live/create API endpoints

#### Related Issue

Closes #4864 

#### Motivation and Context

In order to create live algorithms in the cloud via local lean for each brokerage we support we need these settings classes to standardize the information being sent to the web API. 

#### Requires Documentation Change

N/A

#### How Has This Been Tested?

Tests for all three brokerage additions have been added and ran

#### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactor (non-breaking change which improves implementation)
- [ ] Performance (non-breaking change which improves performance. Please add associated performance test and results)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Non-functional change (xml comments/documentation/etc)

#### Checklist:

- [x] My code follows the code style of this project.
- [x] I have read the **CONTRIBUTING** [document](https://github.com/QuantConnect/Lean/blob/master/CONTRIBUTING.md).
- [x] I have added tests to cover my changes. 
- [x] All new and existing tests passed.
- [x] My branch follows the naming convention `bug-<issue#>-<description>` or `feature-<issue#>-<description>`

### CODE REVIEW

File: Tests/Api/ApiLiveTradingTests.cs [hunk 1]
```diff
@@ -151,6 +151,71 @@ public void LiveTradierTest()
...
+            var key = Config.Get("alpaca-key-id");
+            var secretKey = Config.Get("alpaca-secret-key");
+            var environment = Config.Get("alpaca-trading-mode").ToLowerInvariant().Equals("live") ? BrokerageEnvironment.Live : BrokerageEnvironment.Paper;
+
+            // Create default algorithm settings
+            var settings = new AlpacaLiveAlgorithmSettings(key, secretKey, environment);
+
+            var file = new ProjectFile
+            {
+                Name = "main.cs",
```
Comment (gsalaz98) [inline Tests/Api/ApiLiveTradingTests.cs:190]: I think the `m` in main.cs should be capitalized

File: Tests/Api/ApiLiveTradingTests.cs [hunk 2]
```diff
@@ -151,6 +151,71 @@ public void LiveTradierTest()
...
+            var key = Config.Get("gdax-api-key");
+            var secretKey = Config.Get("gdax-api-secret");
+            var passphrase = Config.Get("gdax-passphrase");
+
+            // Create default algorithm settings
+            var settings = new GDAXLiveAlgorithmSettings(key, secretKey, passphrase);
+
+            var file = new ProjectFile
+            {
+                Name = "main.cs",
```
Comment (gsalaz98) [inline Tests/Api/ApiLiveTradingTests.cs:212]: Capitalize the `m` here as well 👍

**Review by gsalaz98 — COMMENTED** [review]
Just a few minor changes, I think it'd probably work as is but still want to keep parity with the cloud environment.

**Comment by C-SELLERS** [comment]
Updated all new and old tests with a capital 'M' for Main.cs

**Review by gsalaz98 — APPROVED** [review]

**Review by Martin-Molinero — APPROVED** [review]

**Pull request merged**